lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -150,7 +150,11 @@ def is_update_required(self, original, proposed, optional_ignore=None):
 
     def get_orgs(self):
         """Downloads all organizations for a user."""
-        return self.request('/organizations', method='GET')
+        response = self.request('/organizations', method='GET')
+        if self.status == 200:
+            return response
+        else:
```

Comment:
So we tend to not do the else if the previous if-block returns from the function. The else is unneeded.

Reply:
Looking at this again, I would prefer even this:
```python
if self.status != 200:
    self.fail_json(msg='Organization lookup failed')

return response
```

So that the default flow returns the response, except if the status is not 200. I think that's more clear, exceptions go into if-blocks, default flow is to succeed.